README.md

```diff
@@ -2,21 +2,33 @@
 
 # Water Security
 
-### About the project
+## About the project
 The current climate change scenario predicts that almost half of the world's population will live in areas of high water stress by 2050 with limited access to fresh clean water. Governments, national, and international institutions, as well as water management companies, are looking for solutions that can address this growing global water demand. Cities are encouraged to take action on water security, to build resilience to water scarcity and manage this finite resource for the future. 
 
-Based on financial, educational, environmental and demographical data, this project aims to display and predict the water security risks around the world. In order to do so, a Regression Machine Learning pipeline is deployed per risk category (e.g. risk of higher water prices or risk of declining water quality), and a forecast of that risk's severity is made. The regression model is based on engineered and selected features from the data mentioned above.
+Based on financial, educational, environmental and demographical data, this project aims to display and predict the water security risks around the world. In order to do so, a Regression Machine Learning pipeline is deployed per risk category (e.g. risk of higher water prices or risk of declining water quality), and a forecast of that risk's severity is made. The regression model is based on engineered and selected features from the data mentioned above. The project results in an app wich can be deployed following the instructions below.
 
-### Documentation
-The documentation of the project can be found [here](https://mdaiceland.github.io/WaterSecurity/).
+## Documentation
+The documentation of the project can be found [here](https://mdaiceland.github.io/WaterSecurity/). 
 
-### Development and Local Deployment
+The documentation can be regenerated by running the following command:
+``` bash
+python generate_documentation.py
+```
 
-Installing:
-`pip install -r requirements.txt`
+## Development and Local Deployment
 
-Running the Web App:
-`python run.py`
+### Installing
+To install the required packages run the following command:
+```
+pip install -r requirements.txt
+```
 
-Generating Documentation:
-`python generate_documentation.py`
+#### Installing on windows
```

Comment:
What about Mac?

Reply:
I'm still figuring out if it works out of the box, the old mac I used was a lost case.. I'm trying it on a newer one now